velox/connectors/hive/HiveConnector.cpp

```diff
@@ -763,6 +764,51 @@ HiveConnector::HiveConnector(
           std::make_unique<FileHandleGenerator>(std::move(properties))),
       executor_(executor) {}
 
+std::unique_ptr<core::PartitionFunction> HivePartitionFunctionSpec::create(
+    int numPartitions) const {
+  return std::make_unique<velox::connector::hive::HivePartitionFunction>(
+      numBuckets_, bucketToPartition_, channels_, constValues_);
+}
+
+folly::dynamic HivePartitionFunctionSpec::serialize() const {
+  folly::dynamic obj = folly::dynamic::object;
+  obj["name"] = "HivePartitionFunctionSpec";
+  obj["numBuckets"] = ISerializable::serialize(numBuckets_);
+  obj["bucketToPartition"] = ISerializable::serialize(bucketToPartition_);
+  obj["keys"] = ISerializable::serialize(channels_);
+  std::vector<velox::core::ConstantTypedExpr> constValues;
+  constValues.reserve(constValues_.size());
+  for (const auto& value : constValues_) {
```

Comment:
Why do we need to copy constValues_? Can we call serialize(constValues_) directly?